region/brinstar/green/Brinstar Pre-Map Room.json

```diff
@@ -453,7 +453,9 @@
           "nodes": [2],
           "mustStayPut": false
         }},
-        {"refill": ["Missile", "Super"]}
+        {"partialRefill": {"type": "Energy", "limit": 240}},
+        {"partialRefill": {"type": "Missile", "limit": 20}},
+        {"partialRefill": {"type": "Super", "limit": 4}}
```

Comment:
Eventually it could be nice to model this differently - as number of killable enemies or something, and then the refill could take into account if the room is resettable and how much patience there is.